pkg/apis/management.cattle.io/v3/catalog_types.go

```diff
@@ -64,6 +66,7 @@ type CatalogCondition struct {
 	Message string `json:"message,omitempty"`
 }
 
+// Deprecated: CatalogStatus.HelmVersionCommits has been deprecated, which is the only consumer of this type.
```

Comment:
Same as: https://github.com/rancher/rancher/pull/32992#discussion_r645145209
Leaving this open for reviewer's thoughts.

Reply:
Resolved by: https://github.com/rancher/rancher/pull/32992/files#r647739404